src/pypowsybl_jupyter/networkexplorer.py

```diff
@@ -92,6 +106,16 @@ def update_sld_diagram(kv: bool = False):
             else:
                 update_sld(sld_widget, sld_diagram_data, keep_viewbox=kv, enable_callbacks=True)
 
+    def update_map():
+        nonlocal map_widget
+        if sel_ctx.get_selected() is not None:
+            if map_widget==None:
+                map_widget=NetworkMapWidget(network, use_name=use_name, nominal_voltages_top_tiers_filter = nominal_voltages_top_tiers_filter)
```

Comment:
it would be nice to have the parameter `use_line_extensions` available in the explorer too.